src/comet_llm/config.py

```diff
@@ -39,6 +39,9 @@ def _extend_comet_ml_config() -> None:
         "comet.logging.console": {"type": str, "default": "INFO"},
         "comet.raise_exceptions_on_error": {"type": int, "default": 0},
         "comet.internal.check_tls_certificate": {"type": bool, "default": True},
+        "comet.offline": {"type": bool, "default": False},
```

Comment:
```suggestion
        "comet.online": {"type": bool, "default": True},
```
Let's use a similar name and default value as `comet_ml.start` so we are consistent